frontend/dockerfile/dockerfile_lint_test.go

```diff
@@ -546,6 +546,19 @@ COPY Dockerfile /foo
 COPY Dockerfile /bar
 `)
 	checkLinterWarnings(t, sb, &lintTestParams{Dockerfile: dockerfile})
+
+	dockerfile = []byte(`
+FROM scratch
+RUN -<<'EOT'
+env
+EOT
+`)
+	checkLinterWarnings(t, sb, &lintTestParams{
+		Dockerfile:        dockerfile,
+		BuildErrLocation:  4,
+		StreamBuildErr:    "failed to solve: dockerfile parse error on line 4: ENV requires at least one argument",
+		UnmarshalBuildErr: "dockerfile parse error on line 4: ENV requires at least one argument",
```

Comment:
fyi @jedevc

Reply:
SGTM, but if this is a common typo, potentially there should be a more dedicated err message.